docs/operations/data-center/process.sql

```diff
@@ -0,0 +1,53 @@
+-- Create a Table for the captured tenant occupancy events
+CREATE TABLE tenant_occupancy (
+  tenant_id VARCHAR PRIMARY KEY,
+  customer_id BIGINT
+) WITH (
+  kafka_topic='tenant-occupancy',
+  partitions=3,
+  key_format='JSON',
+  value_format='JSON'
+);
+
+-- Create a Stream for the power control panel telemetry data.
+--   tenant_kwh_usage is reset by the device every month
+CREATE STREAM panel_power_readings (
+  panel_id BIGINT,
+  tenant_id VARCHAR,
+  panel_current_utilization DOUBLE,
+  tenant_kwh_usage BIGINT
+) WITH (
+  kafka_topic='panel-readings',
+  partitions=3,
+  key_format='JSON',
+  value_format='JSON'
+);
+
+-- Create a filtered Stream of panel readings registering power usage >= 85%
+--  good for determining panels which are drawing a high electrical load
+CREATE STREAM overloaded_panels AS 
+  SELECT panel_id, tenant_id, panel_current_utilization 
+    FROM PANEL_POWER_READINGS 
```

Comment:
Consistency with below?

```suggestion
    FROM panel_power_readings 
```